core/src/avm2/globals/flash/desktop/NativeApplication.as

```diff
@@ -0,0 +1,264 @@
+package flash.desktop
+{
+  import flash.display.NativeWindow;
+  import flash.events.InvokeEvent;
+  import flash.events.Event;
+  import flash.events.TimerEvent;
+  import flash.events.EventDispatcher;
+  import flash.system.Security;
+  import flash.utils.Timer;
+  import flash.utils.setTimeout;
+  import __ruffle__.stub_method;
+  import __ruffle__.stub_getter;
+  import __ruffle__.stub_setter;
+
+  [API("661")]
+  public final class NativeApplication extends EventDispatcher
+  {
+    private static var _instance:NativeApplication;
+
+    public var _openedWindows:Array = [];
+
+    private var _idleThreshold:int = 300;
+
+    public function NativeApplication()
+    {
+      super();
+      setTimeout(function():void
+      {
+        dispatchEvent(new InvokeEvent(InvokeEvent.INVOKE, false, false, null, []));
+      }, 500);
```

Comment:
I'm assuming this is fake behavior - can you add some comment here?

Reply:
with #17609, `[]` will be replaced by the real arguments passed to the application, but it stay fake behaviour I think

Reply:
I don't know when INVOKE is triggered in AIR so I put a delay in case, so there are no issues.

Reply:
> InvokeEvent objects are dispatched by the NativeApplication object (NativeApplication.nativeApplication). **To receive invoke events, call the addEventListener() method of the NativeApplication object. When an event listener registers for an invoke event, it will also receive all invoke events that occurred before the registration.**

And in airglobal I can see an `override public function addEventListener` for NativeApplication. If Dofus doesn't need this, can we remove it (or at least add a TODO comment) so we can properly implement it later?